packages/client/components/Dashboard/DashSidebar.tsx

```diff
@@ -66,7 +66,7 @@ const DashSidebar = (props: Props) => {
         <Contents>
           <NavItemsWrap>
             <NavItem icon={'forum'} href={'/meetings'} label={'Meetings'} />
-            <NavItem icon={'timeline'} href={'/me'} label={'Timeline'} />
+            <NavItem icon={'timeline'} href={'/me'} label={'History'} />
```

Comment:
@Dschoordsch I'm having a hard time understanding what `icon={'timeline'}` does here. I am importing the history icon a the top, but if I change it to `icon={'history'}` the icon doesn't show up. 

Would you know why?

Reply:
Yes, `NavItem` is just a styled version of [`LeftDashNavItem`](https://github.com/ParabolInc/parabol/blob/4c8bdbe2606549847e6d9cc91ba8f29c9d538868/packages/client/components/Dashboard/DashSidebar.tsx#L43). If you look at the [`LeftDashNavItem` props](https://github.com/ParabolInc/parabol/blob/4c8bdbe2606549847e6d9cc91ba8f29c9d538868/packages/client/components/Dashboard/LeftDashNavItem.tsx#L61) you can see that `icon` is a string. So it's happening inside `LeftDashNavItem` where we map the string to an actual icon.